### DISCUSSION TITLE
feature: add a map key for CopilotChatStop

### DISCUSSION THREAD

**Discussion by chmouel** [LazyVim/LazyVim#3755] [Extra Requests]
Sometime CopilotChat can be verbose and it would be nice to quickly stop it to generate text with a keybinding to the :CopilotChatStop command

I would suggest C-c but i am not sure if it conflicts with something else...

**Comment by dpetka2001** [comment]
I'd advise against `<C-c>`, since there seem to be some issues with Neovim (see [here](https://github.com/neovim/neovim/issues/20726)).
